src/components/common/AgreementType.vue

```diff
@@ -21,43 +21,45 @@
       </div>
     </template>
 
-    <!-- EDIT SECTION -->
-    <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
-      <v-checkbox
-        v-for="(item, index) in getIncorporationAgreementDocuments"
-        v-model="agreementType"
-        class="ml-6 mt-0 agreement-option-list"
-        :id="`agreement-type-${item.code}`"
-        :key="index"
-        :value="item.code"
-        @change="changeAgreementType()"
-      >
-        <template v-slot:label>
-          <div v-html="item.description" class="ml-6 py-4 agreement-option" />
-        </template>
-      </v-checkbox>
-    </div>
-
-    <!-- EDIT SECTION -->
-    <div class="py-8 px-6" v-else :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
-      <v-radio-group
-        class="mt-0 pt-0"
-        hide-details
-        v-model="agreementType"
-        @change="changeAgreementType()"
-      >
-        <v-radio
+    <template v-if="!isSummary">
+      <!-- EDIT SECTION -->
+      <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
+        <v-checkbox
           v-for="(item, index) in getIncorporationAgreementDocuments"
+          v-model="agreementType"
+          class="ml-6 mt-0 agreement-option-list"
+          :id="`agreement-type-${item.code}`"
           :key="index"
           :value="item.code"
-          :id="`agreement-type-${item.code}`"
+          @change="changeAgreementType()"
         >
           <template v-slot:label>
-            <div v-html="item.description" class="agreement-option ml-6" />
+            <div v-html="item.description" class="ml-6 py-4 agreement-option" />
           </template>
-        </v-radio>
-      </v-radio-group>
-    </div>
+        </v-checkbox>
+      </div>
+
+      <!-- EDIT SECTION -->
```

Comment:
This comment doesn't make sense any more.

By the way, I had to look at this a couple of times before I realized that the above section is checkboxes but the section below is radio buttons.

Reply:
I think ULC and CC have only one template so check box but other s have option to choose from